test/test_local_db.js

```diff
@@ -344,11 +373,5 @@ exports['test local and remote'] = function (done) {
     testparseMongoConnectionString,
     testReplicaIncorrectPassword,
     testparseMongoConnectionString
-    ],
-    function (err) {
```

Comment:
Just passing done here will do the trick since the `mocha` callback asserts no error is passed.